web/src/test/java/org/springframework/security/web/authentication/AbstractAuthenticationTargetUrlRequestHandlerTests.java

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Now that we are in 2023, will you please also update the Copyright header?

Reply:
Done.